frontend/src/components/campAdmin/CampActivityProgressLabels.vue

```diff
@@ -3,156 +3,128 @@
     <slot name="title">
       <div class="ec-content-group__title py-1 subtitle-1">
         {{ $tc('components.campAdmin.campActivityProgressLabels.title') }}
-        <dialog-activity-progress-label-create v-if="!disabled" :camp="camp()">
+        <DialogActivityProgressLabelCreate v-if="!disabled" :camp="camp()">
           <template #activator="{ on }">
-            <button-add
+            <ButtonAdd
               color="secondary"
               text
               :hide-label="$vuetify.breakpoint.xsOnly"
               class="my-n2"
               v-on="on"
             >
               {{ $tc('components.campAdmin.campActivityProgressLabels.create') }}
-            </button-add>
+            </ButtonAdd>
           </template>
-        </dialog-activity-progress-label-create>
+        </DialogActivityProgressLabelCreate>
       </div>
     </slot>
-    <v-skeleton-loader v-if="loading" type="article" />
```

Comment:
why did you remove the loading indicator here?

Reply:
Got lost, reimplemented in 90d9e25